docker-compose.yml

```diff
@@ -0,0 +1,53 @@
+version: "3.8"
+
+services:
+  charterafrica:
+    depends_on:
+      mongodb_container:
+        condition: service_started
+    build:
+      context: .
+      dockerfile: Dockerfile.charterafrica
+      args:
+        MONGO_URL: mongodb://root:rootpassword@host.docker.internal:27017/charterafrica?authSource=admin&directConnection=true
+        PAYLOAD_SECRET_KEY: ${PAYLOAD_SECRET_KEY}
+        SENTRY_AUTH_TOKEN: ${SENTRY_AUTH_TOKEN}
+        SENTRY_ORG: ${SENTRY_ORG}
+        NEXT_PUBLIC_SENTRY_DSN: ${CHARTERAFRICA_SENTRY_DSN}
+        SENTRY_ENV: ${SENTRY_ENV}
+        SENTRY_PROJECT: ${SENTRY_PROJECT}
+    ports:
+      - 3000:3000
+    container_name: charterafrica
+    env_file:
+      - apps/charterafrica/.env.local
```

Comment:
Won't the app run without this? Otherwise, we need to update the documentation that there must be a `.env.local` file during development.

Reply:
Yes, the app won't build without env vars such as PAYLOAD_SECRET_KEY and SENTRY variables